app/components/edit/bulk_participants_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Edit
```

Comment:
This is just a move with a little tweak (adding `form_component` as an argument).